portfolio/src/main/webapp/backpacking.html

```diff
@@ -6,19 +6,144 @@
     <link rel="stylesheet" href="style.css">
```

Comment:
Should this change be in this PR? Looks familiar

Reply:
Won't happen again